src/client/unity.yml

```diff
@@ -23,6 +23,60 @@ paths:
             application/json:
               schema:
                 $ref: '#/components/schemas/Error'
+  '/billing/date':
+    get:
+      operationId: GetBillingDate
+      tags:
+        - BillingDate
+      responses:
+        '200':
+          description: Billing Date for Account
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/BillingDate'
+        default:
+          description: Unexpected error
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/Error'
+  '/billing/region':
+    get:
+      operationId: GetRegion
+      tags:
+        - Region
+      responses:
+        '200':
+          description: Account region for billing
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/Region'
+        default:
+          description: Unexpected error
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/Error'
+  '/account/history':
+    get:
+      operationId: GetHistory
+      tags:
+        - History
+      responses:
+        '200':
+          description: Account history for billing, rates and usage
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/History'
+        default:
+          description: Unexpected error
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/Error'
```

Comment:
should this go with the other `/account` endpoint?

Reply:
You bring up a good point. Truthfully I'm not entirely sold on keeping this in the long run since the `history` endpoint in Quartz currently makes a flux request based on certain user meta data (found in quartz) and the selected time range. The big "gotcha" that I have with straight moving the flux query to the UI right now would be because the flux queries query a specific bucket's aggregate data that Quartz is being written to. Figured I'd want to talk to the e-commerce team to try and see where it best fits.

My reasoning for leaving this API separate from the `/billing/account` one is because I'm not sure how large the CSVs being returned from the API would be, so I don't want to overfetch data that isn't being used and potentially slow the browser down if we're getting a couple of 28mb csvs returned.